CoroutineTemplate/app/build.gradle.kts

```diff
@@ -94,6 +106,10 @@ android {
     }
 }
 
+jacoco {
+    toolVersion = "0.8.7"
```

Comment:
Hmmmm, interesting.

Do you know we didn't require this in `RxJavaTemplate`? 🙏